_data/harnesses/inventory/ksql-test.yml

```diff
@@ -0,0 +1,54 @@
+test:
+  steps:
+    - title: ksqlDB code
+      content:
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            skip: true
+
+        - action: execute_async
+          file: docker-compose-up.sh
+          render:
+            skip: true
+
+        - action: execute
+          file: wait-for-containers.sh
+          render:
+            skip: true
+
+        - action: docker_ksql_cli_session
+          container: ksqldb-cli
+          docker_bootup_file: start-cli.sh
+          stdout:
+            directory: tutorial-steps/dev/outputs
+          column_width: 75
+          render:
+            skip: true
+          stdin:
+            - file: ../../confluent/code/tutorial-steps/dev/process.sql
+              render:
+                file: tutorials/inventory/confluent/markup/dev/process.adoc
```

Comment:
FWIW this file render is unnecessary but I couldn't make the harness work without erroring out if I removed this or replaced it with `skip: true`.  If anyone has ideas, please let me know.

Reply:
Update: verified I can remove it completely.  Will be pushing a commit shortly